atintegrators/AperturePass.c

```diff
@@ -65,7 +67,7 @@ void mexFunction(	int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
         }
     }
     else {
-        mexErrMsgIdAndTxt("AT:WrongArg","Needs 0 or 2 arguments");
+        mexErrMsgIdAndTxt("AT:WrongArg","Needs 0, 2 or 3 arguments");

```

Comment:
Why did you change this field? nrhs == 3 is not implemented....

Reply:
To avoid confusion, I would even keep `AperturePass.c` unchanged. This was probably an attempt to create a general function for the 3 planes, but I do prefer the present option to have a separate element and integrators for the longitudinal aperture.

Reply:
Indeed, my mistake. I did try to create a function for all three planes but decided to split it in two functions instead. I forgot to change the error message.